### ISSUE TITLE
Icon is Mis-aligned

### ISSUE THREAD

**Issue by dkt2** [AlaskaAirlines/auro-datepicker#177]
## Describe the bug

The icon is not aligned and I am on the most recent version. 

![Screenshot 2023-12-15 at 11 45 25 AM](https://github.com/AlaskaAirlines/auro-datepicker/assets/16234891/a24d6eb6-4974-4dc7-bd47-07603ffdfb03)

Not sure if this is helpful (or correct), but spelunking into the CSS seems to indicate that removing `top: 6px` seems to bring it back into alignment:

![Screenshot 2023-12-15 at 11 55 52 AM](https://github.com/AlaskaAirlines/auro-datepicker/assets/16234891/846975f7-e64e-405f-b4ae-23d0832eca07)

## To Reproduce

1. Import `auro-datepicker` into a React component
1. Display it

## Expected behavior

For the icon to not be pushed to the top of the datepicker.

## Screenshots

Please reference the screenshots above.

## Desktop (please complete the following information):

 - OS: MacOS Apple M1 Pro
 - Browser Chrome
 - Version Version 119.0.6045.123 (Official Build) (arm64)

**Comment by blackfalcon** [comment]
@dkt2 can you validate this issue in the Auro doc site? I am not sure why you are seeing the calendar icon float to the top. Here is a screenshot from the search UI in AS.com

<img width="479" alt="Screen Shot 2023-12-20 at 4 41 04 PM" src="https://github.com/AlaskaAirlines/auro-datepicker/assets/181089/d5a14553-f3b5-47fb-9e77-75f07ff86561">

<br><br><br><br><br>And here is what I see in the doc site. 

<img width="482" alt="Screen Shot 2023-12-20 at 4 40 13 PM" src="https://github.com/AlaskaAirlines/auro-datepicker/assets/181089/af66fbe2-dbc9-4c62-abd8-2d685a116658">

**Comment by dkt2** [comment]
Hi @blackfalcon, I'm not able to reproduce it on the doc site; would an Auro Engineer be available for a pair on this (in the new year)?

**Comment by blackfalcon** [comment]
@Patrick-Daly-AA can you work with @jordanjones243 and @pgm369 to schedule a pairing session with @dkt2 to review this UI in his environment? 

Since the UI issue is not reproducible either locally or in the docsite this may require an Auro dev to walk through their inspector to discover the issue.

**Comment by blackfalcon** [comment]
@jason-capsule42, @jordanjones243, @dkt2, and I got onto a call to validate the issue. The issue itself stemmed from a bug discovered with a recent update to Token references and fallback values in the element's CSS. 

To resolve @dkt2 issue, we worked with him to update dependencies and code to resolve the UI issue. We also created a new issue and upcoming PR to address the incorrect fallback reference in the code that caused the UI issue in the first place. 

No further actions are needed.
